pre_commit/languages/conda.py

```diff
@@ -0,0 +1,31 @@
+from pre_commit.languages import helpers
+from pre_commit.util import clean_path_on_failure
+from pre_commit.util import cmd_output_b
+
+ENVIRONMENT_DIR = 'conda'
+get_default_version = helpers.basic_get_default_version
+healthy = helpers.basic_healthy
+
+
+def install_environment(prefix, version, additional_dependencies):
+    additional_dependencies = list(additional_dependencies)
+    directory = helpers.environment_dir(ENVIRONMENT_DIR, version)
```

Comment:
`language_version` isn't currently being handled -- there's a helper to show that's not supported: https://github.com/pre-commit/pre-commit/blob/023b337ff0f5566f366dd85c390804208a60f7e7/pre_commit/languages/helpers.py#L27-L31

Reply:
I think this is the only outstanding comment here

Reply:
Ops, didn't see this. Now also fixed.